src/test/java/HomeWork/NotificationMessage.java

```diff
@@ -10,9 +10,8 @@ public class NotificationMessage extends PreCondition {
     @Test
     public void notificationMessage() {
         driver.navigate().to("https://the-internet.herokuapp.com/notification_message_rendered");
-        driver.findElement(By.cssSelector("#content > div > p > a")).click();
-        List<WebElement> Rows = driver.findElements(By.cssSelector("#dropdown > option"));
-        Assert.assertEquals(Rows.size(), 1);
-        Assert.assertEquals(Rows.get(0).getAttribute("outerText"), " Action successful\n×");
+        driver.findElement(By.xpath("//*[@id=\"content\"]/div/p/a")).click();
+        WebElement row = (WebElement) driver.findElement(By.cssSelector("div[class='flash notice']"));
```

Comment:
Why do you need a type casting to (WebElement) here?  
driver.findElement() returns a WeElement y default